lang/java/avro/src/main/java/org/apache/avro/reflect/ReflectData.java

```diff
@@ -110,14 +112,96 @@ public static ReflectData get() {
   }
 
   /**
-   * Cause a class to be treated as though it had an {@link Stringable}
-   ** annotation.
+   * Cause a class to be treated as though it had an {@link Stringable} *
+   * annotation.
    */
   public ReflectData addStringable(Class c) {
     stringableClasses.add(c);
     return this;
   }
 
+  /**
+   * If this flag is set to true, default values for fields will be assigned
+   * dynamically using Java reflections. Let's call this feature `default
```

Comment:
We might say a bit more about how this works, that, when enabled, defaults are the field values of an instance created with a no-arg constructor.

Reply:
I fixed